### PR TITLE
Project 2 News site Caroline Byström

### PR DESCRIPTION
**Pull request by Frivridna**
Finished the project and hopefully reached somewhere between the blue/red stretch goals. 

I have put things I've struggled with in the README file.

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,188 @@
...
+	}
+
+	.nav-item:hover {
+		color:blue;
+	}
+
+	.button {
+		margin-right: 10px;
+		margin-top: 5px;
+	}
```
[the remark this answers is not in the document]
Comment (HIPPIEKICK) [reply code/style.css:48]: Now I see that they are actually outside the width of the HTML element, meaning your content becomes too wide:

Skärmavbild 2021-02-03 kl  14 58 56

File: code/style.css [hunk 2]
```diff
@@ -1,4 +1,188 @@
...
-  color: #fff;
+background: #FFF; 
 }
+
+body {
+	margin: 0;
+}
+
+	.container {
+		width: 320px;
```
Comment (HIPPIEKICK) [inline code/style.css:14]: Just a heads up that this `width` will limit the width of all content:
Skärmavbild 2021-02-03 kl  15 03 22

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Hi Caroline! Good job with this project, I like that you played around with hover effects and the `transition` property 👍 However, I would like to ask you to go through the indentation of  your CSS file as there are many tabs there right now. The idea is that we want VSC to "translate" our tabs to (two) spaces, I made a comment on what your settings should look like. Change this and take a look at the [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) if you're unsure on how the indentation of the CSS file should look.

So to be approved:
- change indentation
- get rid of horizontal scroll caused by too wide content (the buttons in the navbar). Check the inspector if you need.

We’ve also put together some general feedback that applies to many projects this first week. Check it out and take the tips with you to upcoming projects! 👉 [General Feedback - week 2](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)

File: code/style.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (Frivridna) [reply code/style.css]: Hello Matilda! Thank you for your comments. I finally got around to make changes in this project :) One question: I set a height to 100vh but in some media views (iPad, iphone x) you can not see the entire length of all content on the page - can I change this somehow?

**Review by HIPPIEKICK — APPROVED** [review]

**Pull request closed without merging**